node_modules/oae-admin/tenants/tenants.html

```diff
@@ -19,6 +19,7 @@
                 {else}
                     <th>__MSG__HOST__</th>
                 {/if}
+                <th>__MSG__EMAIL_DOMAIN__</th>
```

Comment:
I had considered that, and it wouldn't work because we don't have configuration validation. We need to ensure email domains don't conflict.